README.md

```diff
@@ -33,7 +33,7 @@ Continuous integration
 
 Build Platform           | Status
 ------------------------ | ------------------------------------------------------------------------------------------------- |
-Ubuntu                   | [![Status][ci-ubuntu-18.04]][ci-latest-build] <br> [![Status][ci-ubuntu-20.04]][ci-latest-build]                              <br> [![Status][ci-ubuntu-21.10]][ci-latest-build]                                                |
+Ubuntu                   | [![Status][ci-ubuntu-18.04]][ci-latest-build] <br> [![Status][ci-ubuntu-20.04]][ci-latest-build] <br> [![Status][ci-ubuntu-20.10]][ci-latest-build] <br> [![Status][ci-ubuntu-22.04]][ci-latest-build] |
```

Comment:
Something went wrong here. We don't need to add a new status badge, just change 21.10 to 22.04